application/src/ext-test/java/org/opentripplanner/ext/emission/EmissionTest.java

```diff
@@ -34,10 +34,18 @@
 import org.opentripplanner.transit.model.timetable.Trip;
 import org.opentripplanner.transit.model.timetable.TripTimesFactory;
 
-class EmissionsTest {
+/**
+ * @deprecated The purpose of this test is not clear?
+ * - Is it an integration test, it does notload data from file.
+ * - Or, is it a unit-test on the {@link EmissionItineraryDecorator}? The test package/name does not
+ *   reflect this and there is realy no need to create a
```

Comment:
```suggestion
 *   reflect this and there is really no need to create a
```

A what?

Reply:
This test is deleted in #6628 - it does not focus on one thing, and everything is tested in other unit-tests.